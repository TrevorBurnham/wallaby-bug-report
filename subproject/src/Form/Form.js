import React, { Component } from 'react';

export default class Form extends Component {
  render() {
    return <form className={this.props.className} />;
  }
}
