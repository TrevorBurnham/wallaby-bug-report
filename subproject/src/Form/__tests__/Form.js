import Form from '../Form';
import React from 'react';
import { shallow } from 'enzyme';

describe('Form', () => {
  let tree;
  beforeEach(() => {
    tree = shallow(
      <Form className="custom">
        <input />
      </Form>
    );
  })

  it('passes className through', () => {
    expect(tree.hasClass('custom')).toBe(true)
  });

  it('matches the snapshot', () => {
    expect(tree).toMatchSnapshot();
  });
});
