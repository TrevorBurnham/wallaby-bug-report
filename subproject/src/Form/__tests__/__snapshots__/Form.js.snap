// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Form matches the snapshot 1`] = `
<form
  className="custom"
/>
`;
